Koans/AboutEnumerations.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using DotNetCoreKoans.Engine;
+using Xunit;
+
+namespace DotNetCoreKoans.Koans
+{
+      public class AboutEnumerations : Koan
+      {
+            /*
+            An enumeration is a value type that represents a set of named constants
+            whose underlying type is an integral numeric. You create an enumeration
+            using the enum keywoard and a title followed by the enum's members.
+
+            Following are three definitions of enums that we will be using througout
+            the Koans. Review them first and notice how they're defined, what the
+            structure and syntax looks like, how they are cased, how they are assigned
+            values and then move on to the first step.
+            */
+            enum Planet
```

Comment:
The theme of the koans is zen. Let's avoid mixing in other themes.

Reply:
You're probably right, the Star Wars theme is tired. I'm not exactly sure I know what you mean though about a zen theme, something a little more abstract. I saw examples of Dogs and Chihuahuas and wasn't sure.

Reply:
I could change `Planet` over to `Shape`, might be less distracting.

Reply:
Oh sorry. The idea is that the project uses [Koans](https://en.wikipedia.org/wiki/Koan). Which are a story, dialogue, question, or statement which is used in Zen practice to provoke the "great doubt" and to practice or test a student's progress in Zen. 

Maybe a Koan enumeration with values for story, dialogue, question, and statement.

Reply:
Thank you for the reference. I changed the enum to something a little more zen-like. I left the other two enums as is because I feel like they represent the behavior I'm trying to show, but will be happy to change those as well if it takes away from the overall theme too much.